### DISCUSSION TITLE
Local time does not match - Timestamp Issue

### DISCUSSION THREAD

**Discussion by Hansi-cross** [iotaledger/firefly#8632] [Help & Support]
Hello,
ive got a problem with open my wallet. i reset and backuped erything. Now the following notification appears.
![2024-12-03 12_51_49Fehlkermeldung Firefly](https://github.com/user-attachments/assets/433a9739-d0fe-4505-ba2d-81f67d29de03)
Hope somebody can help.

BR
Hansi

**Comment by begonaalvarezd** [comment]
You have to sync your machine clock since its out of sync, just sync it manually or set it to automatic
